test/streaming/streamingClient.test.ts

```diff
@@ -44,22 +44,26 @@ describe('Streaming API Client', () => {
 
   beforeEach(async () => {
     sandboxStub = createSandbox();
+    await $$.stubAuths(testData);
     // Stub retrieveMaxApiVersion to get over "Domain Not Found: The org cannot be found" error
```

Comment:
Should this file be reverted given streaming client is not sensitive to api version